.github/workflows/typespec-latest-validation.yml

```diff
@@ -0,0 +1,94 @@
+name: TypeSpec Latest Validation
+
+on:
+  issue_comment:
+    types: [created]
+  workflow_dispatch:
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  typespec-latest-validation:
+    name: Validate against TypeSpec Latest
+    runs-on: ubuntu-latest
+    if: |
+      (github.event_name == 'workflow_dispatch') ||
+      (github.event_name == 'issue_comment' && 
+       github.event.issue.pull_request &&
+       contains(github.event.comment.body, '/integration azure-specs') &&
+       (github.event.comment.author_association == 'MEMBER' || 
+        github.event.comment.author_association == 'OWNER' || 
+        github.event.comment.author_association == 'COLLABORATOR'))
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          submodules: true
+          fetch-depth: 0
+          token: ${{ github.token }}
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+
+      - name: Checkout Azure REST API specs
+        uses: actions/checkout@v4
+        with:
+          repository: Azure/azure-rest-api-specs
+          path: azure-rest-api-specs
+          fetch-depth: 1
+
+      - name: Setup Node.js and pnpm
+        uses: ./.github/actions/setup
+
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+
+      - name: Run TypeSpec validation in azure-rest-api-specs
```

Comment:
A better approach would be to first create the tgz for all packages and update the package.json in the azure spec repo to point to those and run npm install on it